src/credentials/FabricTable.h

```diff
@@ -898,7 +942,7 @@ class DLL_EXPORT FabricTable
     CHIP_ERROR AddNewFabricForTestIgnoringCollisions(const ByteSpan & rootCert, const ByteSpan & icacCert, const ByteSpan & nocCert,
```

Comment:
Followup: We should remove this API and push the call to `PermitCollidingFabrics` into the tests, then have them call `AddNewFabricForTest`.

Reply:
#21501